### PR TITLE
Enable SSL cert and update examples with demo certificate.

### PR DESCRIPTION
**Pull request by kowshikn**
*Description of changes:*

Enable SSL cert and also in the examples use the demo certs from ODFE.

Note: I will check with ODFE-Security team and find out if its okay put the [demo certificate](https://github.com/opendistro-for-elasticsearch/security/blob/771199ff191032a75fef3f3bdf33733e49783767/tools/install_demo_configuration.sh#L318-L343).

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by erosas — APPROVED** [review]

File: situp-plugins/elasticsearch/src/main/java/com/amazon/situp/plugins/sink/elasticsearch/ConnectionConfiguration.java
Comment (chenqi0805) [inline situp-plugins/elasticsearch/src/main/java/com/amazon/situp/plugins/sink/elasticsearch/ConnectionConfiguration.java:154]: Just recalled that one thing I have not figured out is whether the hostname verification will still work for a single CA if there are multiple nodes, node-0, node-1 that have different host urls.

**Review by chenqi0805 — APPROVED** [review]

**Comment by kowshikn** [comment]
Added folks from odfe-security team to the PR.

**Review by chenqi0805 — APPROVED** [review]

**Comment by kowshikn** [comment]
Merging this, will still wait for feedback for odfe-security team.

**Pull request merged**
